packages/analytics-core/src/plugins/destination.ts

```diff
@@ -220,7 +204,6 @@ export class Destination implements DestinationPlugin {
       default: {
         // log intermediate event status before retry
         this.config.loggerProvider.warn(`{code: 0, error: "Status '${status}' provided for ${list.length} events"}`);
-        this.handleOtherResponse(list);
```

Comment:
Why are we removing this?

Reply:
This is what we do for handle other response:
```
  handleOtherResponse(list: Context[]) {
    const later = list.map((context) => {
      context.timeout = context.attempts * this.retryTimeout;
      return context;
    });

    const tryable = this.getTryableList(later);
    this.scheduleTryable(tryable);
  }
```
This logic is:
1. setup the timeout for a single event
2. put the tryable event back

Right now we don't need those two, because 
1. schedule event based on a single event timeout will mess up the event order
2. the current logic is leaving those events in the queue and retrying later with the next trigger. 

So it's not needed anymore.

Reply:
Nice! Thanks for simplifying this logic! :)